function.k

```diff
@@ -434,15 +529,15 @@ rule <k> pcsContractPart(C:Id, function F:Id (P1:FParameters) .FunQuantifiers
            <newStack>  .List </newStack>
           
 
-      //rule <k> createTransaction(.List) => . ... </k> 
-         /*  <newStack> L </newStack>
-           requires L =/=K .List*/
+      rule <k> createTransaction(.List) => . ... </k> 
+           <newStack> L </newStack>
+           requires L =/=K .List
```

Comment:
Good! That's the kind of side-condition you want for `L` to be non-empty. 
It's easy to forget sometimes, so make sure you add it when needed!